modules/acquia_cms_headless/composer.json

```diff
@@ -5,7 +5,7 @@
     "license": "GPL-2.0-or-later",
     "require": {
         "drupal/acquia_cms_tour": "2.x-dev",
-        "drupal/jsonapi_extras": "^3.23",
+        "drupal/jsonapi_extras": "3.23",
```

Comment:
@chandan-singh7929  This needs to be added for acquia_cms_component module as well.

Reply:
Thanks @vishalkhode1 I have updated it.